ldap/servers/plugins/replication/cl5_api.c

```diff
@@ -1016,6 +1016,20 @@ cl5GetState()
     return s_cl5Desc.dbState;
 }
 
+void
+cl5ConfigSetCompaction(int compactInterval, char *compactTime)
```

Comment:
Should we also change 'cl5ConfigTrimming' function in cl5_api.c file accordingly?

Reply:
absolutely right. Sorry for that. Fixiing it

Reply:
New patch pushed